lbrynet/conf.py

```diff
@@ -534,8 +534,7 @@ class Config(CLIConfig):
         ('lbrynet4.lbry.com', 4444)  # ASIA
     ])
 
-    # comment server uses a single string because requests are made to the /api resource
-    comment_server = String("Server to store user-generated metadata.", "http://comments.lbry.com:2903/api")
+    comment_server = String("Updated Comment Server version.", "https://comments.lbry.com/api")
```

Comment:
Description of the `comment_server` is not helpful. It sounds like you're supposed to set the version of an updated comment server... What does "updated" even mean in this context, how does it describe what the field is for? Please fix so that someone running `lbrynet start --help` can make sense of what the heck this is for.

Also, can't this field use the `Server` conf type like all the other fields that point to servers? Why does it have to be a string type?